foundation/vault/vault.go

```diff
@@ -150,6 +172,151 @@ func (v *Vault) PublicKeyPEM(kid string) (string, error) {
 
 // =============================================================================
 
+func (v *Vault) Init(ctx context.Context, opts *InitRequest) (*InitResponse, error) {
+	url := fmt.Sprintf("%s/v1/sys/init", v.address)
+
+	var b bytes.Buffer
+	if err := json.NewEncoder(&b).Encode(opts); err != nil {
+		return nil, fmt.Errorf("encode data: %w", err)
+	}
+
+	req, err := http.NewRequestWithContext(ctx, http.MethodPut, url, &b)
+	if err != nil {
+		return nil, fmt.Errorf("create request: %w", err)
+	}
+
+	req.Header.Set("X-Vault-Token", v.token)
+	req.Header.Set("Cache-Control", "no-cache")
+	req.Header.Set("Content-Type", "application/json")
+	req.Header.Set("Accept", "application/json")
+
+	resp, err := v.client.Do(req)
+	if err != nil {
+		return nil, fmt.Errorf("do: %w", err)
+	}
+	defer resp.Body.Close()
+
+	if resp.StatusCode != http.StatusOK {
+		return nil, fmt.Errorf("status code: %s", resp.Status)
+	}
+
+	var response InitResponse
+
```

Comment:
remove this line space and keep the variable declaration agaisnt the code that is using it.